notebooks/257-stable-diffusion-torchdynamo-backend/257-stable-diffusion-torchdynamo-backend.ipynb

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    import openvino.torch  # noqa: F401
</pre><p>I think it is necessary say that for activating openvino backend for torchdynamo it required to import openvino.torch. Possibly better to move this line to the same place where you call torch.compile for consistency and transperency of usage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>